Feature/release v4 sakshi

saksjain opened the pull request.

saksjain committed:
Updated code for contacts

updated the code and folder for advisories so that contacts (mets + health) to remove duplication.

saksjain committed:
added the advisory app to the existing wildfire shiny app

saksjain committed:
updated the names of the new files to match with the existing files

saksjain committed:
added copyright info

saksjain committed:
fixed ENV contact issue + changed name to "community smoke"

saksjain committed:
fixed logos and added title/type to YAML header

saksjain committed:
updated the issue file according to the template

dawnbcgov committed:
add cards for contacts and update links to align on

dawnbcgov committed:
update logo generation to match wildfire templates

dawnbcgov committed:
update variable name

dawnbcgov committed:
add in separators for contacts

saksjain committed:
Merge pull request #71 from bcgov/feat/add-shortcodes

add cards for contacts and update logos to align with wildfire warnings

saksjain committed:
added customText

the default text is for issue-advisory. The textbox and YAML have been added for end-advisory but no custom text will show up in end-advisory in the .md and .pdf files.

saksjain committed:
edited the title, formatted mandatory reductions and burn ban section

saksjain committed:
added functionality for multiple pollutants for issue #64

saksjain committed:
updated style for health tips (#72), edited title and text for continue advisory

saksjain committed:
made some formatting changes

saksjain committed:
added custom text box for burn ban area and end date/time

GailR reviewed (commented):
Hi Sakshi,

I think we are getting close. Below are some high-levels suggestions with detailed notes in the inline comments.

**From a functional perspective** 
- Remove BC Sans font from the YAML (in the `format: pdf:` sections). This causes the Shiny.io version to fail.
- Review the YAML parameters -- see inline comments for specifics
- Consider splitting `module_community_smoke` into two modules. One for issuing and one for ending warnings. This will help tailor the UI to each use case, reducing the cognitive load by only showing relevant input fields. 
- Add accordions and cards for markdown output for a consistent presentation across all warning types.

**Formatting and streamlining (lower priority)**
- suggested naming updates have been provided for consistency and clarity
- Consider merging the `src` and `utilities` directories - they seem to have overlapping purposes

Let me know once the functional updates are complete.  I'll test in Shiny.io and the `aqwarning` repo.

saksjain commented on community_smoke_end.qmd:
```diff
@@ -1,17 +1,42 @@
-  station: Whistler
+  sel_aqMet: "Sakshi Jain"
+  station: "Whistler"
   nextBulletin: "2025-05-08"
```
I removed the parameter altogether and added the code for the next day directly in the markdown.

saksjain commented:
To do: 

- [ ] Fix the formatting for "mu" in both issue and end advisory for pdf format.
- [ ] Add accordions and cards.
- [ ] NextBulletin - discuss. Is this a shiny parameter?
- [ ] Files in directory src and utility - merge into one?

saksjain committed:
addressed gail's comments

#73 - a few comments left to address.

saksjain committed:
updated end advisory to match the template

saksjain committed:
added NextUpdate shiny feature & merged "src" and "utility" files

DonnaHaga committed:
commented out customMessageBanArea in end module to stop app from crashing

GailR committed:
- adjusted YAML title to exclude open burning restrictions (no option for user to include/exclude); discuss with team if option to include/exclude is necessary for end warning
- updated YAML parameters to ensure necessary parameters are captured in .md output for GitHub pages

GailR committed:
- simplified YAML title
- updated YAML parameters to ensure necessary parameters are captured in .md output for GitHub pages
